aws/inferentia/build.gradle

```diff
@@ -8,13 +8,16 @@ version = '0.0.1-SNAPSHOT'
 
 repositories {
     mavenCentral()
+    maven {
+        url 'https://oss.sonatype.org/content/repositories/snapshots/'
+    }
 }
 
 dependencies {
-    implementation platform("ai.djl:bom:0.10.0")
+    implementation platform("ai.djl:bom:0.12.0-SNAPSHOT")
```

Comment:
If you are using 0.12.0-SNAPSHOT, can we just use the built-in environment variable to load the extra files?

Reply:
I commented out manually loading libneuron_op.so in the example code.